File: python/8 kyu/90.py
# https://www.codewars.com/kata/57cff961eca260b71900008f/train/python

# Passed

def is_vow(inp):
    vowels = {'a', 'e', 'i', 'o', 'u'}
    return [char_code if chr(char_code) not in vowels else chr(char_code) for char_code in inp]

output = is_vow([100,100,116,105,117,121])
print(output)

File: nodejs/7 kyu/90.js
// https://www.codewars.com/kata/5ace2d9f307eb29430000092/train/javascript

// Passed

function modifyMultiply(str, loc, num) {
  const words = str.split(" ");
  const wordToMultiply = words[loc];
  let resultArray = [];
  for (let i = 0; i < num; i++) {
    resultArray.push(wordToMultiply);
  }
  const result = resultArray.join("-");
  return result;
}

const output = modifyMultiply(
  "Creativity is the process of having original ideas that have value. It is a process; it's not random.",
  8,
  10
);
console.log(output);